templates/config.edn

```diff
@@ -1,137 +1,160 @@
 
- ;; Specify the date on which the week starts.
- ;; Goes from 0 to 6 (Monday to Sunday), default to 6
+ ;; Specify the first day of the week.
+ ;; Default value: 6 (Sunday)
+ ;; Available options:
+ ;;  - integer from 0 to 6 (Monday to Sunday)
  :start-of-week 6
 
- ;; Specify a custom CSS import
- ;; This option take precedence over your local `logseq/custom.css` file
- ;; You may find a list of awesome logseq themes here:
- ;; https://github.com/logseq/awesome-logseq#css-themes
- ;; Example:
- ;; :custom-css-url "@import url('https://cdn.jsdelivr.net/gh/dracula/logseq@master/custom.css');"
+ ;; Import a custom CSS file.
+ ;; This option takes precedence over the local `logseq/custom.css` file.
+ ;; :custom-css-url "@import url('');"
 
- ;; Specify a custom js import
- ;; This option take precedence over your local `logseq/custom.js` file
+ ;; Import a custom JS file.
+ ;; This option takes precedence over the local `logseq/custom.js` file.
  ;; :custom-js-url ""
 
  ;; Set a custom Arweave gateway
  ;; Default gateway: https://arweave.net
  ;; :arweave/gateway ""
 
  ;; Set Bullet indentation when exporting
- ;; default option: tab
- ;; Possible options for `:export/bullet-indentation` are
- ;; 1. `:eight-spaces` as eight spaces
- ;; 2. `:four-spaces` as four spaces
- ;; 3. `:two-spaces` as two spaces
- ;; :export/bullet-indentation :tab
-
- ;; When :all-pages-public? true, export repo would export all pages within that repo.
- ;; Regardless of whether you've set any page to public or not.
- ;; Example:
- ;; :publishing/all-pages-public? true
-
- ;; Specify default home page and sidebar status for Logseq
- ;; If not specified, Logseq default opens journals page on startup
- ;; value for `:page` is name of page
- ;; Possible options for `:sidebar` are
- ;; 1. `"Contents"` to open up `Contents` in sidebar by default
- ;; 2. `page name` to open up some page in sidebar
- ;; 3. Or multiple pages in an array ["Contents" "Page A" "Page B"]
- ;; If `:sidebar` is not set, sidebar will be hidden
- ;; Example:
- ;; 1. Setup page "Changelog" as home page and "Contents" in sidebar
+ ;; Default value: tab
+ ;; Available options:
+ ;;  - `:eight-spaces` as eight spaces
+ ;;  - `:four-spaces` as four spaces
+ ;;  - `:two-spaces` as two spaces
+ :export/bullet-indentation :tab
+
+ ;; Publish all pages within the Graph
+ ;; regardless of whether individual pages have been marked as public.
+ ;; Default value: false
+ :publishing/all-pages-public? false
+
+ ;; Set the default home page and sidebar status.
+ ;; If not specified, Logseq opens the journal page on startup by default.
+ ;; The `:page` value is the name of the page.
+ ;; Possible options for :sidebar include:
+ ;; 1. "Contents" to display Contents in the right sidebar by default.
+ ;; 2. page name to display a specific page in the right sidebar.
+ ;; 3. An array of multiple pages, e.g., ["Contents" "Page A" "Page B"].
+ ;; If `:sidebar` is not set, the right sidebar will be hidden.
+ ;; Examples:
```

Comment:
```suggestion
 ;; If `:sidebar` is not set, the right sidebar will be hidden.
 ;;
 ;; Examples:
```
I think a newline in the comment here helps break up instructions vs examples, and makes it clear the examples are for the whole thing, not the sidebar line.